Dnn.AdminExperience/ClientSide/SiteSettings.Web/src/components/privacySettings/index.jsx

```diff
@@ -208,33 +208,6 @@ class PrivacySettingsPanelBody extends Component {
           />
         </InputGroup>
       </div>
-    ) : (
-      <div key="column-one-left" className="left-column" />
-    );
```

Comment:
I _think_ we want to keep this `column-one-left` and remove the `column-one-right` below.

Reply:
I sent a commit to resolve this one.